hypermap/aggregator/tasks.py

```diff
@@ -288,6 +288,19 @@ def index_layer(self, layer_id, use_cache=False):
 
 
 @shared_task(bind=True)
+def unindex_layers_with_issues(self, use_cache=False):
+    """
+    Remove the index for layers in search backend, which are linked to an issue.
+    """
+    from hypermap.aggregator.models import Issue, Layer
+    from django.contrib.contenttypes.models import ContentType
+
+    layer_type = ContentType.objects.get_for_model(Layer)
```

Comment:
I think we need also to run the code for all of the layers which are part of a service which an associated issue